src/Widgets/ThumbnailGalleryImageWidget/ThumbnailGalleryImageWidgetEditingConfig.js

```diff
@@ -17,6 +17,23 @@ Scrivito.provideEditingConfig("ThumbnailGalleryImageWidget", {
     },
   },
   properties: ["image", "title", "subtitle", "tags"],
-  titleForContent: widget =>
-    `${widget.get("title")} - ${widget.get("subtitle")}`,
+  titleForContent: widget => {
+    const parts = [widget.get("title"), widget.get("subtitle")];
+    const summary = parts.filter(e => e).join(" - ");
+
+    if (summary) {
+      return `Thumbnail Gallery Image: ${summary}`;
+    }
+  },
+  validations: [
+    [
+      "image",
+
+      image => {
+        if (!image) {
+          return { message: "The image should be set.", severity: "warning" };
+        }
```

Comment:
Drop it (just the thumbnail gallery validation is enough)

Reply:
👍